webpack/__tests__/ForemanRhCloudTestHelpers.test.js

```diff
@@ -0,0 +1,12 @@
+import { testSelectorsSnapshotWithFixtures } from 'react-redux-test-utils';
+import { inventoryStateWrapper } from '../ForemanRhCloudTestHelpers';
+
+const fixtures = {
+  'should return inventory wrapper': () =>
+    inventoryStateWrapper({ inventoryChildren: null }),
+};
+
+global.URL_PREFIX = '';
```

Comment:
it's not needed here I think,
can you move it to https://github.com/theforeman/foreman_rh_cloud/pull/201/files#diff-c13550dd3f83e3bc665bd6ce480cc1f8R3

Reply:
can you remove this line?